src/runtime/Makefile

```diff
@@ -261,6 +265,12 @@ DEFGUESTPREATTESTATIONSECRETTYPE ?= connection
 DEFSEVCERTCHAIN ?= /opt/sev/cert_chain.cert
 DEFSEVGUESTPOLICY ?= 0
 
+# SNP
+DEFSNPGUEST ?= true
+# Based on SEV Secure Nested Paging Firmware ABI Specification section 4.3
+# unspecified or == 0 --> 0x30000 i.e. Bit#17 is '1' and Bit#16 is '1' (SMT is allowed)
+DEFSNPGUESTPOLICY ?= 0xB0000
```

Comment:
This comment is a bit confusing to me and I thought we wanted the policy to be `0x30000`.